README.md

```diff
@@ -2,102 +2,102 @@
-- [A day described in Grakn](https://towardsdatascience.com/a-day-described-in-grakn-part-i-46bb7e3cd82a)
-- [Introducing GraMi — a Data Migration Tool for Grakn](https://towardsdatascience.com/introducing-grami-a-data-migration-tool-for-grakn-d4051582f867)
-- [Comparing Graql to SQL — Part 1/2](https://dzone.com/articles/comparing-graql-to-sql-part-12)
-- [Comparing Grakn to Semantic Web Technologies — Part 1/3](https://towardsdatascience.com/comparing-grakn-to-semantic-web-technologies-part-1-3-3558c447214a)
-- [Comparing Grakn to Semantic Web Technologies — Part 2/3](https://towardsdatascience.com/comparing-grakn-to-semantic-web-technologies-part-2-3-4602b56969fc)
-- [Comparing Grakn to Semantic Web Technologies — Part 3/3](https://towardsdatascience.com/comparing-grakn-to-semantic-web-technologies-part-3-3-cb2fb233fbd5)
-- [Neo4j vs GRAKN Part I: Basics](https://towardsdatascience.com/neo4j-vs-grakn-part-i-basics-f2fe3511ce88)
-- [Neo4j vs GRAKN Part II: Semantics](https://towardsdatascience.com/neo4j-vs-grakn-part-ii-semantics-11a0847ae7a2)
 
 ## Videos
 
-- [Causal Knowledge Graphs Using TypeDB](https://www.youtube.com/watch?v=pqcZ9_4-79U)
+- [Introduction to TypeDB and TypeQL](https://www.youtube.com/watch?v=e0lmTSb-rzY)
+- [Pandora Paper Leaks With TypeDB](https://www.youtube.com/watch?v=9EaxwUG9vAg)
+- [Building a Distributed Database With the Raft Replication Algorithm](https://www.youtube.com/watch?v=oS42lN4qSJ4)
+- [Enabling the Computation Future of Biology](https://www.youtube.com/watch?v=XJDr_prOp9g)
 - [Ontology-Driven Modelling for Text-Mining Around Technology and Engineering](https://www.youtube.com/watch?v=0ir12x5FB4s)
 - [Using TypeDB for Content Authorisation Across the Enterprise](https://www.youtube.com/watch?v=H-Ln8YUITRw)
-- [Comparing Graph Databases to TypeDB](https://www.youtube.com/watch?v=JTxvJxVNSH4)
-- [Comparing SQL to Graql](https://www.youtube.com/watch?v=rNCvAGdiw1I&list=PLtEF8_xCPklbGuaOANOJHZ8cvbo65LN2I)
-- [Comparing Semantic Web Technologies To Grakn | RDF, RDFS, OWL, SPARQL SHACL Grakn and Graql](https://www.youtube.com/watch?v=s6xGQS84qKM)
-- [Comparing Property Graph Databases to Grakn](https://www.youtube.com/watch?v=uxZovj2ekIQ)
+- [Comparing Graph Databases to TypeDB | What you can't do with graphs](https://www.youtube.com/watch?v=JTxvJxVNSH4)
+- [Comparing SQL to TypeQL](https://www.youtube.com/watch?v=uhPDohXVcjU)
+- [Comparing Semantic Web Technologies To TypeDB](https://www.youtube.com/watch?v=LFgV7sCnOrE)
 - [How Can We Complete a Knowledge Graph?](https://www.youtube.com/watch?v=nYDi1_UaFtU&list=PLtEF8_xCPklbGuaOANOJHZ8cvbo65LN2I)
 
+## Client Drivers (official)
+
+List of officially supported drivers to connect with TypeDB Server.
+
+- [`client-java`](https://github.com/vaticle/typedb-client-java)
+- [`client-nodejs`](https://github.com/vaticle/typedb-client-nodejs)
+- [`client-python`](https://github.com/vaticle/typedb-client-python)
```

Comment:
Even though it's not finished yet, I would add Rust ahead of time, because it's definitely the most awesome of them all (and definitely something we might forget to add to the awesome list even once it's done):
```suggestion
- [Client Rust](https://github.com/vaticle/typedb-client-rust)
- [Client Java](https://github.com/vaticle/typedb-client-java)
- [Client Node.js](https://github.com/vaticle/typedb-client-nodejs)
- [Client Python](https://github.com/vaticle/typedb-client-python)
```